### PR TITLE
mrc-3901 dockerise tool

### PR DESCRIPTION
**Pull request by hillalex**
At the moment just used in OW build for testing: https://github.com/vimc/orderly-web/pull/502/files#diff-c7dddfd1bb4a15878f71537f08f14360593f422522a40e9b5137cc1cf0a8c338R15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,18 @@ To install `outpack.orderly`:
 remotes::install_github("mrc-ide/outpack.orderly", upgrade = FALSE)
 ```
 
+## Usage from docker
+
+Pass the `src` and `destination` directories as command line arguments.
+Note that the `dest` directory must not exist, so you have to mount a parent directory 
+into the container and specify the desired dest as a subdirectory:
+```
+docker run -v /orderly/path:/orderly \
```
Comment (richfitz) [inline README.md:25]: which of these can be readonly? just the orderly one? why `parent`?

File: docker/entrypoint.R [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env Rscript
+
+options <- commandArgs(trailingOnly = TRUE)
+outpack.orderly::orderly2outpack(options[[1]], options[[2]])
```
Comment (richfitz) [inline docker/entrypoint.R:4]: this entrypoint does the migration once only, then exits? do we want a container that stays up and runs on some schedule?

Comment (richfitz) [reply docker/entrypoint.R:4]: Probably worth using docopt here really, too

**Review by richfitz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hillalex) [reply README.md:25]: why parent - see comment on line 22, but maybe the better thing to do would be change the behaviour here!

File: docker/entrypoint.R (hunk 2 above)
Comment (hillalex) [reply docker/entrypoint.R:4]: also considered just relying on mountpoints (i.e. mount orderly to /orderly, outpack to /outpack and then use those?)

Comment (hillalex) [reply docker/entrypoint.R:4]: this container is just used to bring up an instance of outpack metadata for testing the proxying functionality in OW. I guess if we bring docopt in and have a fuller interface we can have args for specifying one-off vs scheduled migrations

**Review by richfitz — APPROVED** [review]

**Pull request merged**